### PR TITLE
Adding changelog for release

### PR DESCRIPTION
**Pull request by RussTheAerialist**
Prepping for the release.  Version number was already incremented.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## 0.2 (02 November 2016)
+
+* Added Initial Schedules Support (#48)
+* Added Initial Create Group endpoint (#69)
+* Added Connection Credentials for publishing datasources (#80)
+* Added Pager object for handling pagination results and sample (#72, #90)
+* Added ServerInfo endpoint (#84)
+* Code Cleanup
```
Comment (t8y8) [inline CHANGELOG.md]: It's a good thing that we're at the point of development where we don't list every little cleanup, right? 🎉

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## 0.2 (02 November 2016)
+
+* Added Initial Schedules Support (#48)
+* Added Initial Create Group endpoint (#69)
+* Added Connection Credentials for publishing datasources (#80)
```
Comment (t8y8) [inline CHANGELOG.md]: His changes apply to workbooks as well

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## 0.2 (02 November 2016)
```
Comment (t8y8) [inline CHANGELOG.md]: Note about deprecating `site` in favor of `site_id`?

File: CHANGELOG.md (hunk 1 above)
Comment (RussTheAerialist) [inline CHANGELOG.md]: Changelogs are mostly for end users.  Calling out that we cleaned up code is nice, but the details of that is less important to end users I think.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,9 +2,10 @@
 
 * Added Initial Schedules Support (#48)
 * Added Initial Create Group endpoint (#69)
-* Added Connection Credentials for publishing datasources (#80)
+* Added Connection Credentials for publishing datasources/workbooks (#80)
 * Added Pager object for handling pagination results and sample (#72, #90)
 * Added ServerInfo endpoint (#84)
+* Deprecated `site_id` as a parameter to `TableauAuth` in favor of `site`
```
Comment (t8y8) [inline CHANGELOG.md]: Opposite, right?

`site` was deprecated and you should use `site_id`

**Pull request merged**